docs/insert_sql_userguide.md

```diff
@@ -0,0 +1,125 @@
+
+    // select data to write
+    val df = spark.sql("select * from tpch_test.ORDERS")
+   ```
+
+## TiDB Options
+
+The following table shows the TiDB-specific options, which can be passed in through `TiDBOptions` or `SparkConf`.
+
+| Key                         | Required | Default                | Description                                                                                                              |
+| --------------------------- |----------| ---------------------- |--------------------------------------------------------------------------------------------------------------------------|
+| pd.addresses                | true     | -                      | The addresses of PD clusters, split by comma                                                                             |
+| database                    | true     | -                      | TiDB Database                                                                                                            |
+| table                       | true     | -                      | TiDB Table                                                                                                               |
+| tidb.addr                   | false    | -                      | TiDB address, needed when enableUpdateTableStatistics is true                                                            |
+| tidb.port                   | false    | -                      | TiDB Port, needed when enableUpdateTableStatistics is true                                                               |
+| tidb.user                   | false    | -                      | TiDB User, needed when enableUpdateTableStatistics is true                                                               |
+| tidb.password               | false    | -                      | TiDB Password, needed when enableUpdateTableStatistics is true                                                           |
+| enableRegionSplit           | false    | true                   | To split Region to avoid hot Region during insertion                                                                     |
+| scatterWaitMS               | false    | 300000                 | Max time to wait scatter region                                                                                          |
+| writeThreadPerTask          | false    | 1                      | Thread number each spark task use to write data to TiKV                                                                  |
+| bytesPerRegion              | false    | 100663296 (96M)        | Decrease this parameter to split more regions (increase write concurrency)                                               |
+| enableUpdateTableStatistics | false    | false                  | Update statistics in table `mysql.stats_meta` (`tidb.user` must own update privilege to table `mysql.stats_meta` if set true) |
```

Comment:
- I think you can't pass enableRegionSplit,writeThreadPerTask,bytesPerRegion,scatterWaitMS beacause you don't get them in `newWriteBuilder`.
- database , table and pd.addresses is needn't.
- lack TiDB_ROW_FORMAT_VERSION

Reply:
I have fix point two and three. For point one, we will get them in  newWriteBuilder now.